hydromt_wflow/wflow.py

```diff
@@ -3163,14 +3171,25 @@ def read_staticgeoms(
     def write_geoms(
         self,
         geom_fn: str = "staticgeoms",
+        precision: int = 6,
     ):
         """Write geoms in <root/geom_fn> in GeoJSON format."""
         # to write use self.geoms[var].to_file()
         if not self._write:
             raise IOError("Model opened in read-only mode")
         if self.geoms:
             self.logger.info("Writing model staticgeom to file.")
+            # Set projection to 1 decimal if projected crs
+            if self.crs.is_projected:
+                precision = 1
```

Comment:
You added this since my last review no? Here the user cannot change that value at all the way you now coded it and for models like dflow fm precision of 10cm may not be good enough. Maybe you could instead set precision to None and then if precision is None and crs is projected then precision is 1 (maybe 2 is better?) and else 6? Then the user is fully in control again.

Reply:
It was after your first review, we had a call concerning the precision when talking about degrees (which need quite some precision) and linear units (metres) which need less. I do agree that having it fixed is not ideal. I'll fiddle around with it a bit to make sure users can set it.